src/gui/widgets/memory_observer.cc

```diff
@@ -0,0 +1,222 @@
+/***************************************************************************
+ *   Copyright (C) 2019 PCSX-Redux authors                                 *
```

Comment:
2022 :)

Reply:
Ha, I had a variety of project files open as references. :)